azure/main.go

```diff
@@ -73,10 +73,17 @@ func (az Azure) CreateNode(ctx context.Context, node model.Node, callback func(s
 
 	customData := cloudinit.EncodedCustomData(node, "/dev/sdc")
 
+	sizeForMode := map[string]int{
```

Comment:
should it be in the config or a property of node? I see it used in 4 different places.

Reply:
I'm thinking of moving this to a config package yes. Because these values are valid for all cloud providers.